javascript/webpack.common.babel.js

```diff
@@ -28,7 +28,7 @@ export default {
                 })
             },
             {
-                test: /\.(png|gif|jpe?g)$/,
+                test: /\.(png|gif|jpe?g|svg)$/,
```

Comment:
Please make sure this change gets made in SST, too (you can commit it directly to master).